src/components/find-on-shelf/find-on-shelf.test.ts

```diff
@@ -0,0 +1,70 @@
+describe("Find on shelf modal", () => {
+  beforeEach(() => {
+    // Intercept holdings data call.
+    cy.fixture("material/find-on-shelf-holdings.json")
+      .then((result) => {
+        cy.intercept("GET", "**/external/agencyid/catalog/holdings/**", {
+          statusCode: 200,
+          body: result
+        });
+      })
+      .as("Find on shelf holdings");
+    // Intercept availability data call.
+    cy.fixture("material/find-on-shelf-holdings.json")
+      .then(() => {
+        cy.intercept("GET", "**/external/agencyid/catalog/availability/**", {
+          statusCode: 200,
+          body: [
+            {
+              recordId: "06373674",
+              reservable: true,
+              available: true,
+              reservations: 48
+            }
+          ]
+        });
+      })
+      .as("Find on shelf availability");
+    cy.visit("/iframe.html?id=components-find-on-shelf-modal--periodical");
+    cy.contains("button:visible", "FIND PÅ HYLDEN").click();
```

Comment:
Consider whether the text "FIND PÅ HYLDEN" needs to be in all caps. In general I would expect such stylistic casing to be controlled with CSS.

Consider whether this text "FIND PÅ HYLDEN" should be in English as it has been decided that this is the default language - also for controls in Storybook.

Reply:
Great points! Fixed both ;)